src/RPCServer.ts

```diff
@@ -77,7 +72,7 @@ class RPCServer extends EventTarget {
    * the handler to handle timeout before it is forced to end. Defaults to 2,000 milliseconds.
    * @param obj.logger
    */
-  public static async createRPCServer({
+  public static async startRPCServer({
```

Comment:
We don't use `startRPCServer`. See how `StartStop` is used in `QUICServer`. We use `QUICServer.start`. There is no static function.

Please review `QUICServer` first before asking for another review.

Reply:
@addievo before you resolve a conversation. Put down a comment explaining how you resolved it.